mmengine/fileio/backends/oss_backend.py

```diff
@@ -0,0 +1,517 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+import re
+from pathlib import Path
+from typing import Iterator, Optional, Tuple, Union
+
+import mmengine
+from .base import BaseStorageBackend
+
+
+class OSSBackend(BaseStorageBackend):
```

Comment:
Should this backend be named `OSSBackend` or `AliyunBackend`? @zhouzaida

Reply:
OSSBackend is OK.